website/content/docs/v0.3/Guides/rpi4-as-servers.md

```diff
@@ -0,0 +1,186 @@
+---
+description: "Using Raspberrypi pi 4 as servers"
+weight: 6
+---
+
+# Raspberrypi4 as servers
+
+This guide will explain on how to use Sidero to manage Raspberrypi-4's as
+servers. This guide goes  hand in hand with the [bootstrapping
+guide](/docs/v0.3/guides/bootstrapping).
+
+From the bootstrapping guide, reach "Install Sidero" and come back to this
+guide. Once you finish with this guide, you will need to go back to the
+bootstrapping guide and continue with "Register the servers".
+
+The rest of this guide goes with the assumption that you've a cluster setup with
+sidero and ready to accept servers. This guide will explain the changes that
+needs to be made to be able to accept RPI4 as server.
+
+# RPI4 boot process
+To be able to boot talos on the Pi4 via network, we need to undergo a 2 step
+boot process. The Pi4 has an EEPROM which contains code to boot up the Pi. This
+EEPROM expects a specific boot folder structure as explained on
+[this](https://www.raspberrypi.org/documentation/configuration/boot_folder.md) page.
+We will use the EEPROM to boot into UEFI, which we will then use to PXE and iPXE boot into sidero & talos.
+
+# Prerequisites
+## Update EEPROM
+To enable the EEPROM on the Pi to support network booting, we must update it to
+the latest version. Visit the
+[release](https://github.com/raspberrypi/rpi-eeprom/releases) page and grab the
```

Comment:
This should probably mention that the EEPROM image here is slightly different than the one mentioned in Talos' RPi4 docs.

Reply:
Added a note, looks good?

Reply:
Yep!